configure-memory-usage.md

```diff
@@ -153,7 +153,7 @@ TiDB 支持对执行算子的数据落盘功能。当 SQL 的内存使用超过
 
 - 落盘行为由参数 [`tidb_mem_quota_query`](/system-variables.md#tidb_mem_quota_query)、[`tidb_enable_tmp_storage_on_oom`](/system-variables.md#tidb_enable_tmp_storage_on_oom)、[`tmp-storage-path`](/tidb-configuration-file.md#tmp-storage-path)、[`tmp-storage-quota`](/tidb-configuration-file.md#tmp-storage-quota) 共同控制。
 - 当落盘被触发时，TiDB 会在日志中打印一条包含关键字 `memory exceeds quota, spill to disk now` 或 `memory exceeds quota, set aggregate mode to spill-mode` 的日志。
-- Sort、MergeJoin、HashJoin 落盘是从 v4.0.0 版本开始引入的，非并发 HashAgg 的落盘是从 v5.2.0 版本开始引入的，并发 HashAgg 的落盘是从 v8.0.0 版本开始引入的。
+- Sort、MergeJoin、HashJoin 落盘是从 v4.0.0 版本开始引入的，非并发 HashAgg 的落盘是从 v5.2.0 版本开始引入的，并发 HashAgg 的落盘是从 v8.0.0 版本开始引入，在 v8.2.0 正式 GA。用户可以通过系统变量`tidb_enable_parallel_hashagg_spill`来关闭和打开并发 HashAgg 的落盘功能(这是临时系统变量，未来会被删除)。
```

Comment:
```suggestion
- Sort、MergeJoin、HashJoin 落盘是从 v4.0.0 版本开始引入的，非并发 HashAgg 的落盘是从 v5.2.0 版本开始引入的，并发 HashAgg 的落盘在 v8.0.0 版本以实验特性引入，在 v8.2.0 版本成为正式功能 (GA)。你可以通过系统变量 [`tidb_enable_parallel_hashagg_spill`](/system-variables.md#tidb_enable_concurrent_hashagg_spill-从-v800-版本开始引入) 控制是否启用支持落盘的并发 HashAgg 算法。该变量将在未来版本中废弃。
```

Remove the warning in L1783-L1885:

https://github.com/pingcap/docs-cn/blob/ce3458ca27e646a6ad6098b8f14c9bfa0755c442/system-variables.md?plain=1#L1781-L1792